proto/tendermint/liquidity/v1beta1/liquidity.proto

```diff
@@ -289,52 +272,7 @@ message PoolBatch {
         }];
 }
 
-message PoolBatchResponse {
-    option (gogoproto.equal) = true;
-    option (gogoproto.goproto_stringer) = true;
-
-    // index of this batch
-    uint64 index = 1 [(gogoproto.moretags) = "yaml:\"index\"",
-        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-            example: "\"1\"",
-            format: "uint64"
-        }];
-
-    // height where this batch is begun
-    int64 begin_height = 2 [(gogoproto.moretags) = "yaml:\"begin_height\"",
-        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-            example: "\"1000\"",
-            format: "int64"
-        }];
-
-    // last index of DepositMsgStates
-    uint64 deposit_msg_index = 3 [(gogoproto.moretags) = "yaml:\"deposit_msg_index\"",
-        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-            example: "\"1\"",
-            format: "uint64"
-        }];
-
-    // last index of WithdrawMsgStates
-    uint64 withdraw_msg_index = 4 [(gogoproto.moretags) = "yaml:\"withdraw_msg_index\"",
-        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-            example: "\"1\"",
-            format: "uint64"
-        }];
-
-    // last index of SwapMsgStates
-    uint64 swap_msg_index = 5 [(gogoproto.moretags) = "yaml:\"swap_msg_index\"",
-        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-            example: "\"1\"",
-            format: "uint64"
-        }];
-
-    // true if executed, false if not executed yet
-    bool executed = 6 [(gogoproto.moretags) = "yaml:\"executed\"",
-        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-            example: "true",
-        }];
-}
-
+// DepositMsgState defines the state of deposit message that contains state information as it is processed in the next batch(s)
```

Comment:
```suggestion
// DepositMsgState defines the state of the deposit message that contains the state information as it is processed in the next batch or batches
```

Reply:
tip: write out singular or plural instead of using parentheses for plural. (sometimes one or more works too, as in "one or more batches") see https://developers.google.com/style/plurals-parentheses